cartography/rules/data/frameworks/mitre_attack/requirements/t1190_exploit_public_facing_application/__init__.py

```diff
@@ -0,0 +1,135 @@
+"""
+MITRE ATT&CK Framework
+Security framework based on MITRE ATT&CK tactics and techniques
+"""
+
+from cartography.rules.spec.model import Fact
+from cartography.rules.spec.model import Module
+from cartography.rules.spec.model import Requirement
+
+# AWS
+_aws_ec2_instance_internet_exposed = Fact(
```

Comment:
using `_` as a prefix convention here to indicate that these queries are not meant to be imported into other files